### PR TITLE
Fix tarball generation.

### PR DESCRIPTION
**Pull request by WilliamZapata**

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go [hunk 1]
```diff
@@ -301,8 +357,20 @@ func generateTarball(hadoopDistribution string) error {
 	run("adding Alluxio server assembly jar", "mv", fmt.Sprintf("assembly/server/target/alluxio-assembly-server-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "assembly", fmt.Sprintf("alluxio-server-%v.jar", version)))
 	run("adding Alluxio FUSE jar", "mv", fmt.Sprintf("integration/fuse/target/alluxio-integration-fuse-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "fuse", fmt.Sprintf("alluxio-fuse-%v.jar", version)))
 	run("adding Alluxio checker jar", "mv", fmt.Sprintf("integration/checker/target/alluxio-checker-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "checker", fmt.Sprintf("alluxio-checker-%v.jar", version)))
-	// Condense the webapp into a single .war file.
-	run("jarring up webapp", "jar", "-cf", filepath.Join(dstPath, webappWar), "-C", webappDir, ".")
+
+	masterWebappBuildDir := "alluxio-ui/master/build"
+	run ("creating alluxio-ui master webapp directory and copying files", "mkdir", "-p", filepath.Join(dstPath, masterWebappBuildDir))
+	err = CopyDir(masterWebappBuildDir, filepath.Join(dstPath, masterWebappBuildDir))
```
Comment (aaudiber) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go]: could this be simplified to one line without need of a help function?
`run ("Moving master webapp resources", "mv", masterWebappBuildDir, filepath.Join(dstPath, masterWebappBuildDir)`

Comment (WilliamZapata) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go]: Yes, I will do, thanks!

Comment (aaudiber) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go]: oops, style should be `run(`, not `run (`

Comment (WilliamZapata) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go]: I got ya

File: docs/_data/table/common-configuration.csv [hunk 2]
```diff
@@ -99,6 +99,7 @@ alluxio.underfs.s3a.streaming.upload.enabled,"false"
 alluxio.underfs.s3a.streaming.upload.partition.size,"64MB"
 alluxio.web.file.info.enabled,"true"
 alluxio.web.resources,"${alluxio.home}/core/server/common/src/main/webapp"
+alluxio.web.resources,"${alluxio.home}/alluxio-ui/"
```
Comment (calvinjia) [inline docs/_data/table/common-configuration.csv]: Do we need to delete the previous line? Also this file is usually autogenerated.

Comment (WilliamZapata) [reply docs/_data/table/common-configuration.csv]: Yes, we do, thank you. Although now that I know it's autogenerated, I will revert this change.

**Pull request merged**